### DISCUSSION TITLE
Snippets in tables

### DISCUSSION THREAD

**Discussion by jimenezk** [squidfunk/mkdocs-material#3325] [Q&A] [answered]
Hi everyone.
Is there a workaround to add snippets in a table? I often want to add content in tables, such as bullets or icons or some sort, and it gets messy by putting it all in one row and adding `<br>`. Not sure if this is a valid use of tables/snippets, or maybe I am abusing their intended use. I appreciate any comments.

So, this works:
```
--8<-- "snips.txt"
```

This doesn't:
```
Column 1 | Column 2
--- | ---
Row 1 | --8<-- "snips.txt"
```

Kevin

**Comment by facelessuser** [comment, marked as answer]
You're going to have to use raw HTML with a combination of the `md_in_html` extension.  Can't recall if you need to add `markdown="1"` on `tr` or `td`, so you'll have to try it out and see:

```html
<table mardown="1">
<tr>
<th>Column 1</th>
<th>Column 2</th>
</tr>
<tr>
<td>Row1</td>
<td>
--8<-- "snips.txt"
</td>
</tr>
</table>
```

**Reply by facelessuser** [reply]
In short, there currently isn't an inline snippet insertion method right now. Snippets is more a block-level type insertion and account for appropriate indentation etc. Inline is a bit tricky because if you inject a file inline, you have to consider how to adjust the content to work in the inline case as indentation and such are very important in Markdown.

**Reply by jimenezk** [reply]
Here we go! That worked. In my `snips.txt` file, this:
```
This is some text file that will be displayed with a snippet.
<br>
Another lines.
```
Is a much more manageable and cleaner (and "scalable") version than one line with the rest of the table contents.

Thanks so much! I can go with this.
